fuzz/src/lib.rs

```diff
@@ -27,7 +27,9 @@ pub fn generate_valid_module(
     config.exceptions_enabled = u.arbitrary()?;
     config.canonicalize_nans = u.arbitrary()?;
     config.tail_call_enabled = u.arbitrary()?;
+
     config.gc_enabled = u.arbitrary()?;
+    config.reference_types_enabled = config.reference_types_enabled || config.gc_enabled;
```

Comment:
Instead of doing this here (and below), could this be configured in wasm-smith's `Arbitrary for Config` implementation? Basically if gc is enabled force-enable reference types.

Reply:
The `Arbitrary for Config` impl always leaves `gc_enabled = false` right now though. Agreed we should do that eventually but it isn't feasible until we want to stop defaulting `gc_enabled = false`.

Reply:
Aha good point! I think I got mixed up a bit too. Given the change here, is the `||` in `validate_valid_module.rs` below necessary?

Reply:
Yeah I guess not.